packages/ddg2dnr/lib/gpc.js

```diff
@@ -1,32 +0,0 @@
-/** @module gpc */
-const {
-    resourceTypes,
-    generateDNRRule
-} = require('./utils')
-
-const GPC_HEADER_PRIORITY = 40000
-
-function generateGPCheaderRules (config) {
```

Comment:
Couldn't this library still have a function for creating the DNR rule, but be agnostic to extension internals like config? i.e. `generateGPCheaderRule(exceptions)`. The extension-side code can then check settings and config then pass the relevant arguments to this function call?